composer.json

```diff
@@ -23,7 +23,8 @@
         "symfony/css-selector": "^2.8|^3.0",
         "symfony/debug-bundle": "^2.8|^3.0",
         "symfony/security": "^2.8|^3.0",
-        "symfony/security-bundle": "^2.8|^3.0"
+        "symfony/security-bundle": "^2.8|^3.0",
+        "symfony/translation": "^2.8|^3.0"
```

Comment:
Loading a component for type hinting is too much IMO, this should be required when you need autoloading in functional/unit tests.

But let's wait for some other opinions :)